pkg/api/deep_copy_generated.go

```diff
@@ -379,7 +379,7 @@ func deepCopy_api_PolicyRule(in authorizationapi.PolicyRule, out *authorizationa
 			if newVal, err := c.DeepCopy(val); err != nil {
 				return err
 			} else {
-				out.Verbs[key] = newVal.(struct{})
+				out.Verbs[key] = newVal.(util.Empty)
```

Comment:
It's an aliased type so this is how just deep copy generation works.  I think upstream we want to have more efficient generations for some contexts (and faster), but will have to wait.

----- Original Message -----
> > @@ -379,7 +379,7 @@ func deepCopy_api_PolicyRule(in
> > authorizationapi.PolicyRule, out *authorizationa
> >  			if newVal, err := c.DeepCopy(val); err != nil {
> >  				return err
> >  			} else {
> > -				out.Verbs[key] = newVal.(struct{})
> > +				out.Verbs[key] = newVal.(util.Empty)
> 
> wouldn't it be cleaner to have a `deepCopy_util_StringSet`?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/3278/files#r32659310